pkg/executor/plugins/docker/executor.go

```diff
@@ -0,0 +1,37 @@
+package docker
+
+import (
+	"fmt"
+	"os/exec"
+
+	"github.com/hashicorp/go-plugin"
+
+	"github.com/bacalhau-project/bacalhau/pkg/executor/plugins"
+)
+
+func NewDockerPlugin() (plugins.PluggableExecutor, error) {
```

Comment:
Instantiates the docker executor plugin (assumes its binary has been build). The `Cmd` feild below informs the client of the shelling point to the plugin. The returned value of this method is a client that speaks RPC to the executor.